lego/apps/users/models.py

```diff
@@ -468,7 +470,7 @@ def restricted_lookup(self):
     def announcement_lookup(self):
         return [self]
 
-    def unanswered_surveys(self):
+    def unanswered_surveys(self) -> list:
```

Comment:
Actually returning `list[Survey]`, but the `Survey` model is imported inside of the function, so this'll have to do.

Reply:
You can forward reference types by making the type hint a string: `"List[Survey]"`

See https://peps.python.org/pep-0484/#the-problem-of-forward-declarations

Reply:
Yea I tried `list["Survey"]`, but my interpreter was behaving weirdly, giving a warning that `"Survey" is not defined`. Will look into.

Reply:
`Pylance` is still telling me that `"Survey" is not defined`, but I can live with that.

Reply:
Actually, `flake8` is now upset, so `list` it is.